File: java/Clothing.java
public class Clothing {
    private String size;
    private String material;
    private String gender;

    // Default constructor
    public Clothing() {
        this.size = "";
        this.material = "";
        this.gender = "";
    }

    // Parameterized constructor
    public Clothing(String size, String material, String gender) {
        this.size = size;
        this.material = material;
        this.gender = gender;
    }

    // Getters
    public String getSize() {
        return size;
    }

    public String getMaterial() {
        return material;
    }

    public String getGender() {
        return gender;
    }

    // Setters
    public void setSize(String size) {
        this.size = size;
    }

    public void setMaterial(String material) {
        this.material = material;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }
}

File: cpp/Clothing.cpp
#include "Clothing.hpp"

// Default constructor
Clothing::Clothing() : size(""), material(""), gender("") {}

// Parameterized constructor
Clothing::Clothing(std::string size, std::string material, std::string gender)
    : size(size), material(material), gender(gender) {}

// Getters
std::string Clothing::getSize() {
    return size;
}

std::string Clothing::getMaterial() {
    return material;
}

std::string Clothing::getGender() {
    return gender;
}

// Setters
void Clothing::setSize(std::string size) {
    this->size = size;
}

void Clothing::setMaterial(std::string material) {
    this->material = material;
}

void Clothing::setGender(std::string gender) {
    this->gender = gender;
}
